CONTRIBUTING.md

```diff
@@ -627,6 +626,7 @@ should be canceled.
 - [Aaron Clawson](https://github.com/MadVikingGod), LightStep
 - [Robert Pająk](https://github.com/pellared), Splunk
 - [Tyler Yahn](https://github.com/MrAlias), Splunk
+- [David Ashpole](https://github.com/dashpole), Google
```

Comment:
I think we have it ordered by surname.

```suggestion
- [David Ashpole](https://github.com/dashpole), Google
- [Aaron Clawson](https://github.com/MadVikingGod), LightStep
- [Robert Pająk](https://github.com/pellared), Splunk
- [Tyler Yahn](https://github.com/MrAlias), Splunk
```

Reply:
I don't think we do, just happens to be that way.  The approvers are not.